spec/amp-var-substitutions.md

```diff
@@ -65,6 +65,37 @@ For instance:
 ```
 may make a request to something like `https://foo.com/pixel?path=%2Fpage1.html`.
 
+**CLIENT_ID**
+
+Use the special string `CLIENT_ID` to add a per document-source-origin (The origin of the website where you publish your AMP doc) and user identifier. The CLIENT_ID will be the same for the same user if they visit again within one year. It should behave roughly similar to a cookie storing a session id for one year. Please take note of the allowed arguments you may
+pass into `CLIENT_ID` like a function. (spaces between arguments and values are not allowed)
+
+arguments:
+  - `cid scope` (Required) - Name of the fallback cookie when the document
+    is not served by an AMP proxy.
+  - `amp-user-notification id` (Optional) - Optionally make the `CLIENT_ID`
+    substitution be dependent on the dismissal of a user notification shown to the visitor
+    of the page.
+
+For instance:
+```html
+<amp-pixel src="https://foo.com/pixel?cid=CLIENT_ID(google-analytics)"></amp-pixel>
+
+
+<amp-user-notification
+    layout=nodisplay
+    id="cookie-consent"
```

Comment:
Please remove `cookie-`. Maybe rename to `user-content`. Cookie is too specific and we can really make any recommendation as to when consent would be needed.